examples/recvlive.cpp

```diff
@@ -70,6 +70,9 @@ int main(int argc, char* argv[])
       return 0;
    }
 
+   // Test the deprecated option feature here:
+   //srt_setsockopt(sfd, 0, SRTO_STRICTENC, &no, sizeof no);
+
    // Windows UDP issue
```

Comment:
It might be worth having it under `ifdef` so that it can be included in CI.

Reply:
No one pays attention for warnings from CI.

This whole mess is intended to make a hint for users that might try to use deprecated options. That's all. In our projects we simply shouldn't use them.

Reply:
Never mind.
Will delete deprecated options in #994 anyway.

Reply:
Yes, but the mechanism should stay.